suite/suite_test.go

```diff
@@ -245,23 +247,29 @@ func (suite *SuiteTester) TestSubtest() {
 		{"first"},
 		{"second"},
 	} {
-		suiteT := suite.T()
+		suite.SuiteT = suite.T()
 		suite.Run(t.testName, func() {
 			// We should get a different *testing.T for subtests, so that
 			// go test recognizes them as proper subtests for output formatting
 			// and running individual subtests
 			subTestT := suite.T()
-			suite.NotEqual(subTestT, suiteT)
+			suite.NotEqual(subTestT, suite.SuiteT)
 		})
-		suite.Equal(suiteT, suite.T())
+		suite.Equal(suite.SuiteT, suite.T())
 	}
 }
 
 func (suite *SuiteTester) TearDownSubTest() {
+	subTestT := suite.T()
+	suite.NotEqual(subTestT, suite.SuiteT)
+
```

Comment:
Hey, something like this would allow you to add your assertions alongside the others and call `assert.Contains` with the specific values we expect to see:
```suggestion
	suite.TearDownSubTestNames = append(suite.SetupSubTestNames, suite.T().Name())
```